src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,42 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
-    # TO-DO
-    
+    merged_arr = []
+
+    l, r = 0, 0
```

Comment:
python is about easy reading of code; try to be descriptive with your variable names